scripts/start-test-brokers.sh

```diff
@@ -2,7 +2,8 @@
 
 THIS_DIR="$(cd "$(dirname "$0")" && pwd)"
 
-cd $THIS_DIR/../_build/default/lib/brod/docker
+git clone git@github.com:klarna/brod.git
```

Comment:
this link requires auth, mb clone via https instead? Or just download a tarball?